blackduck-artifactory-common/src/main/java/com/synopsys/integration/blackduck/artifactory/configuration/ConfigurationPropertyManager.java

```diff
@@ -70,32 +79,41 @@ public Properties getProperties() {
                    .entrySet();
     }
 
+    @Nullable
     public String getProperty(ConfigurationProperty configurationProperty) {
-        return properties.getProperty(configurationProperty.getKey());
+        return getProperty(configurationProperty.getKey());
     }
 
+    @Nullable
     private String getProperty(String propertyKey) {
         return properties.getProperty(propertyKey);
     }
 
+    @Nullable
     public Boolean getBooleanProperty(ConfigurationProperty configurationProperty) {
         return getBooleanProperty(configurationProperty.getKey());
     }
 
+    @Nullable
     private Boolean getBooleanProperty(String propertyKey) {
         return BooleanUtils.toBooleanObject(getProperty(propertyKey));
     }
 
+    @Nullable
     public Integer getIntegerProperty(ConfigurationProperty configurationProperty) {
         return getIntegerProperty(configurationProperty.getKey());
     }
 
+    @Nullable
     private Integer getIntegerProperty(String propertyKey) {
         Integer value = null;
-
         try {
-            value = Integer.valueOf(getProperty(propertyKey));
-        } catch (NumberFormatException e) {
+            String propertyValue = getProperty(propertyKey);
+            if (propertyValue == null) {
+                throw new IntegrationException("");
```

Comment:
I noticed I didn't provide a message here. Added to the 8.0.0 branch with commit fe7b7ecd3e88cee3d63764dde213004c43c00f44